web-src/axinite/src/styles/semantic.css

```
@@ -0,0 +1,2623 @@
+.catalogue-table-wrap {
+  overflow-x: auto;
+}
+
+.catalogue-table__caption {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)